### ISSUE TITLE
rpi2+1.5.1 logi board bitcoin_app never ends

### ISSUE THREAD

**Issue by datajerk** [fpga-logi/logi-projects#22]
Only output:

`[0, 0, 0, 0]`

About one line/5 seconds.  Once on start I got `nonce ffffffff` immediately then exit. 

After a bit of googling I did find a beaglebone reference that said it'd take 2 min.  I've been waiting for hours.

LED1 is blinking ~2 times/sec.

Thanks.

**Comment by jpiat** [comment]
Hi,

did you install the logi-apps for logipi and tested the wishbone app (this
will allow to make sure the communication with the FPGA is working fine) ?

Regards,

Jonathan Piat

2015-08-05 5:27 GMT+02:00 datajerk <notifications@github.com>:

**Comment by datajerk** [comment]
Yes I installed the logipi apps, but only tested LED and Bitcoin app. Led app worked just fine. 

I'll test wishbone next.

**Comment by datajerk** [comment]
Update.  I went back to using stock LOGI PI distro and it works just fine.
So something with kernel 4.x not working correctly (custom image based on
latest debian+kernel).  I'll continue to look into it, but I consider this
matter closed.  Thanks.
